lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -153,7 +153,10 @@ class CategoryRenderer {
    */
   _setRatingClass(element, score, scoreDisplayMode) {
     const rating = Util.calculateRating(score, scoreDisplayMode);
-    element.classList.add(`lh-audit--${rating}`, `lh-audit--${scoreDisplayMode}`);
+    element.classList.add(`lh-audit--${rating}`);
+    if (typeof scoreDisplayMode === 'string') {
```

Comment:
wait...how would it be possible for it not to be? That seems bad if it happens. Just in tests maybe (hopefully)?

Reply:
from https://travis-ci.org/GoogleChrome/lighthouse/builds/470305380#L816, it looks like just the unit tests. We should fix those (is that what fdf9755 did?) and then not change the code here, I think. Better to trust our type system and fail fast if we defined/enforced things poorly.

Reply:
Yeah fdf9755ad113997bdfdb0d476a0827066b921427 fixed the unit tests and added guard rails for if the scoreDisplayMode was undefined.  Good point tho, I removed the guardrails and made all the tests actually represent reality!